packages/gestalt-codemods/generic-codemods/deprecatedAvatarPair.js

```diff
@@ -0,0 +1,34 @@
+/**
```

Comment:
Dont add a custom codemod, just use a generic codemod https://github.com/pinterest/gestalt/blob/master/packages/gestalt-codemods/generic-codemods/detectManualReplacement.js

yarn codemod detectManualReplacement ~/path/to/your/code
 * --component=AvatarPair

Reply:
Add a comment on the description. Look at  https://github.com/pinterest/gestalt/pull/2102